File: Front/src/app/models/reportingActivite.ts
export class ReportingActivite {


  nomActivite: string;
  groupe: string;
  nbEC: number;
  nbV: number;
  nbF: number;
  nbM: number;
  cotisations: number;



  constructor() {

    this.nomActivite = "";
    this.groupe = "";
    this.nbEC = 0;
    this.nbV = 0;
    this.nbF = 0;
    this.nbM = 0;
    this.cotisations = 0;
  }
}

File: Back/src/main/java/com/gestion/adhesion/models/ReportingActivite.java
package com.gestion.adhesion.models;

import lombok.*;

@Getter
@Setter
@ToString
@Builder
@AllArgsConstructor
@RequiredArgsConstructor
public class ReportingActivite {

    private String nomActivite;

    private Long nbInitee =0L;

    private Long nbPayee =0L;

    private Long nbValidee =0L;

    private Long nbF =0L;

    private Long nbM =0L;

    private Long cotisations =0L;

}